doc/fileexplorer.rst

```diff
@@ -1,23 +1,84 @@
+The **Files** pane is a filesystem and directory browser built right into Spyder.
+You can view and filter files according to their type and extension, open them with the :doc:`editor` or an external tool, and perform many common operations.
 
 .. image:: images/files/files-standard.png
-   :alt: Spyder File Explorer panel, showing a tree view of files and metadata
+   :alt: Spyder Files pane, showing a tree view of files and metadata
 
 
 
-==========================
-Performing file operations
-==========================
+===============
+File operations
+===============
 
-You can use context menus to access a number of functions on a per-file and per-directory basis, including running scripts, creating, renaming, moving, editing or deleting files, performing basic operations with ``git``, and opening an external file explorer.
+To browse the files on your system, you can use the arrows at the top of the pane.
+You can expand/collapse the folders in the pane to display the files and subdirectories hierarchically. 
+Double-clicking a folder will open it, showing the files inside and making it your working directory.
 
-.. image:: images/files/files-contextmenu-new.png
-   :align: center
-   :alt: File Explorer context menu, with common file manipulation tasks
+.. image:: images/files/files-browse.gif
+   :alt: Spyder Files pane showing browsing directories
+
+
+To open a file in the :guilabel:`Editor` from the :guilabel:`Files` pane, double-click its name.
+If you right-click over it, you will see a context menu that allows you to access a number of functions, including running scripts; creating, renaming, moving, deleting files; and opening them in your computer's file manager.
+
+.. image:: images/files/files-context-menu.png
+   :alt: Spyder Files pane showing context menu
+
+You can copy and paste one or several files to and from the pane, and copy their absolute or relative paths to the clipboard as text.
+If copying the paths for multiple files, they will be automatically formatted so you can paste them directly into a Python list.
+
+.. image:: images/files/files-copy-path.gif
+   :alt: Spyder Files pane showing copying absolute path to Editor
+
+Additionally, the :guilabel:`Files` pane allows you to perform basic operations with ``git``, like commiting your changes and browsing the repository a given file or folder is part of.
+
+.. image:: images/files/files-commit.gif
+   :alt: Spyder Files pane showing commiting change of file
+
+
+
+============
+Options menu
+============
+
+The options menu in the top right of the :guilabel:`Find` pane offers several ways to customize how your files are displayed.
+
+By default, the option :guilabel:`Show all files` is activated, which displays the contents of your working directory without filtering. 
+However, if you deactivate this, the pane will only show files matching the patterns set under :guilabel:`Show filenames with these extensions...`.
```

Comment:
```suggestion
However, the pane can show only files matching the patterns set under :guilabel:`Show filenames with these extensions...`.
```

Fix phrasing according to my suggestion above.

Reply:
```suggestion
However, it can filter the list to show only files matching the patterns set under :guilabel:`Show filenames with these extensions...`, if you toggle the :guilabel:`Activate filename filters` button in the pane toolbar.
```

Clearer phrasing and actually mention how to enable this, I don't see it anywhere else.

Also, there's a serious bug with the tooltip of the filter toggle in 4.2—its called "Activate filename filters", but the option (as makes much more sense) _enables_ the filters. Therefore, the tooltip needs to be changed to read `Activate filename filters`.